README.md

```diff
@@ -502,6 +503,16 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## Svelte
+
+- [Svelte Official documentation](https://svelte.dev/tutorial/basics)
+- [Svelte for beginners](https://coursetro.com/posts/code/176/Learn-Svelte-3.0---Svelte-Tutorial-for-Beginners)
```

Comment:
As this is a single blog post I think it doesn't qualify it so please remove it.